### DISCUSSION TITLE
OMI glTF Working Group Meeting 2022-11-17

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/gltf-extensions#117] [General]
This meeting will be on 11/17/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Last Week's Agenda for reference:

- OMI_link
- OMI_spawn_point
- We're meeting in a Frame space, check the omi-gltf-extensions channel for details
- glTF Transform Updates
- [OMI_collider](https://github.com/omigroup/gltf-extensions/pull/118)
- [EXT_mesh_lod](https://github.com/takahirox/EXT_mesh_lod)
- [KHR_audio / Khronos feedback](https://github.com/KhronosGroup/glTF/pull/2137)
  - [gltf-sample-assets](https://github.com/KhronosGroup/glTF-Sample-Models/pull/360)
- [glXF](https://github.com/KhronosGroup/glXF/blob/main/specification/2.0/README.md)
  - OMI_id / [OMI_ref](https://github.com/omigroup/gltf-extensions/pull/85)
- USDz spike
- Schedule alternate timezone meeting
- Show and tell
- Create next weeks agenda

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by robertlong** [comment]
No meeting next week due to US Thanksgiving Day

# OMI Collider
- Lets get a final say on the cylinder decision and vote to merge the OMI_collider extension in two weeks if we have quorum.

**Comment by madjin** [comment]
I wanna visualize all the activities / explorations happening in this working group via a booth display / virtual world, some notes

![omi_gltf-extensions-booth](https://user-images.githubusercontent.com/32600939/202583079-9b732451-e38a-4f4d-bfda-f630bbcd9306.jpg)
